x-pack/plugins/snapshot_restore/public/index.scss

```diff
@@ -0,0 +1,13 @@
+// Import the EUI global scope so we can use EUI constants
+@import 'src/legacy/ui/public/styles/_styling_constants';
+
+// Snapshot and restore plugin styles
+
+// Prefix all styles with "snapshot" to avoid conflicts.
```

Comment:
I know there aren't any actual styles in here (yet) but prefixes should just be 3 letters. I would change it now so its correct if people start adding styles.

Reply:
thanks for that info, I've updated this to be `ssr` prefix instead